pkg/3scale/amp/operator/backend_options_provider.go

```diff
@@ -58,6 +58,10 @@ func (o *OperatorBackendOptionsProvider) GetBackendOptions() (*component.Backend
 	o.backendOptions.ListenerMonitoringLabels = o.listenerMonitoringLabels()
 	o.backendOptions.WorkerMonitoringLabels = o.workerMonitoringLabels()
 
+	boolTmp := true
```

Comment:
Why are we enabling this unconditionally? shouldn't be based on the monitoring cr flag?

Reply:
see comments above